arch/arm/soc/atmel_sam0/samd21/dts.fixup

```diff
@@ -1,3 +1,8 @@
+/* SoC level DTS fixup file */
+
+#define FLASH_DEV_BASE_ADDRESS		ATMEL_SAM0_NVMCTRL_41004000_BASE_ADDRESS_0
+#define FLASH_DEV_NAME			ATMEL_SAM0_NVMCTRL_41004000_LABEL
+
 #define CONFIG_GPIO_SAM0_PORTA_LABEL ATMEL_SAM0_GPIO_41004400_LABEL
```

Comment:
Can you pull the dts.fixup move/additions into their own commit in the PR.  I'd like them separated from the board code commit if possible.

Reply:
Done as #6305